app/views/content/docs/sales.md

```diff
@@ -0,0 +1,67 @@
+[Documentation](/docs) &gt; Resources &gt; Sales
+
+## Sales API
+
+Sales and Auctions are available at the sales endpoint.
+
+#### Retrieving Sales
+
+Users can retrieve their own applications by following the [sales](#{ArtsyAPI.artsy_api_root}/sales) link from [root](#{ArtsyAPI.artsy_api_root}).
+
+```
+curl -v "#{ArtsyAPI.artsy_api_root}/sales" -H "X-Access-Token:#{access_token}"
+```
+
+This endpoint accepts the following parameters.
+
+Name       | Value                                  |
+----------:|:---------------------------------------|
+live       | Boolean- filter live auctions only     |
+is_auction | Boolean- filter auctions only          |
+published  | Boolean- filter published auctions only|
+
+``` alert[danger]
+Only authorized users/applications may access **unpublished** sales (where published=false)
```

Comment:
I don't fully understand what you mean here. What does the `published` param do, with and without `live` and `is_auction`?